pyccel/ast/cwrapper.py

```diff
@@ -1244,7 +1303,7 @@ def C_to_Python(c_object):
                     body = [])
 
 # https://docs.python.org/3/c-api/object.html#c.PyObject_GetIter
-PySet_GetIter = FunctionDef(name = 'PyObject_GetIter',
+PyObj_GetIter = FunctionDef(name = 'PyObject_GetIter',
```

Comment:
Why not calling it with the original name `PyObject_GetIter`?

Reply:
No good reason. I just prefer shorter names.
Changed : fe6c6ce87